src/components/fields/location-field/location-helper.ts

```diff
@@ -143,6 +143,28 @@ export namespace LocationHelper {
 		}
 	};
 
+	export const checkAndSetPinLocationAsResult = (query: string): IResultsMetaData => {
+		const [lat, lng] = query
+			.split(":")[1]
+			.split(",")
+			.map((value) => parseFloat(value.trim()));
+		let parsedResult = [];
+		if (LocationHelper.isCoordinateInBounds({ lat: lat, lng: lng })) {
+			parsedResult = [
+				{
+					address: query,
+					lat: lat,
+					lng: lng,
+				},
+			];
+		}
+		return {
+			results: parsedResult,
+			apiPageNum: 1,
+			totalNumPages: 1,
+		};
+	};
+
```

Comment:
In the case where the coordinates provided by the user is out of bounds, the behaviour is no pin location "search result" is produced in the list, so the user cannot confirm the change.